.github/workflows/publish-canary.yml

```
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - develop
+      - release
```

Need to handle dates in the release name

```suggestion
      - release-*
```

---

(discussed on slack)